dashboard/test/models/data_doc_test.rb

```diff
@@ -13,4 +13,18 @@ class DataDocTest < ActiveSupport::TestCase
       DataDoc.create!(key: 'myblock', name: 'invalid block')
     end
   end
+
+  test "serialize returns an object with the correct fields" do
```

Comment:
This is kinda a silly test ... I'm thinking I can replace this by using seeding once that logic is implemented

Reply:
makes sense... running both serialize and seed is a nice way to test things